src/hazelcore/ZTyp.re

```diff
@@ -168,7 +168,8 @@ let move_cursor_right_zoperator: zoperator => option(zoperator) =
 
 let rec move_cursor_right = (zty: t): option(t) =>
   zty |> move_cursor_right_zopseq
-and move_cursor_right_zopseq = zopseq =>
+and move_cursor_right_zopseq = zopseq => {
+  print_endline("ZTyp move_cursor_right_zopseq");
```

Comment:
don't leave debug prints when you mark as ready for review

Reply:
I thought I removed these. My bad.